wolfcrypt/src/sp_int.c

```diff
@@ -4394,10 +4394,20 @@ int sp_init(sp_int* a)
  */
 int sp_init_size(sp_int* a, int size)
 {
-    int err = sp_init(a);
+    int err = MP_OKAY;
+
+    if (a == NULL) {
```

Comment:
Pull common code out into _sp_init().
Have the size passed in.
I don't want to have to find everywhere that say HAVE_WOLF_BIGINT needs to be changed.

Reply:
done